sdk/servicebus/Azure.Messaging.ServiceBus/src/Transports/Amqp/AmqpBodyType.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.ServiceBus.Transports.Amqp
+{
+    /// <summary>
+    /// The types of AMQP bodies for an AMQP message.
+    /// </summary>
+    public enum AmqpBodyType
+    {
+        /// <summary>
+        /// The type of body for the message could not be determined based on the information in the message.
+        /// </summary>
+        Unspecified,
```

Comment:
Do we need an explicit "we don't know" value?   I'm wondering if we assume `Value` when we don't know to match the current (and legacy) semantics of "this is just an opaque blob of bytes ".

Reply:
Yes. This directly maps to Body (both sending and receiving). We consider Body to be a 4th type of AMQP body type, because it needs logic to convert Body to Data and visa versa. Josh and I had a lengthy discussion on this earlier in the week.